Description: Exclude note about program licenses from copyright string
 The directory referenced would include copyright information for any
 programs or libraries that were built by Macaulay2.  However, as we
 use the corresponding Debian packages instead, this directory is
 never created.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-25

--- a/M2/Macaulay2/d/startup.m2.in
+++ b/M2/Macaulay2/d/startup.m2.in
@@ -668,8 +668,6 @@
 
 initcurrentlayout()
 
-copyright = copyright | newline | "For licenses see " | prefixDirectory | currentLayout#"program licenses"
-
 path = (x -> select(x, i -> i =!= null)) deepSplice {
 	  if prefixDirectory =!= null then (
 	       if topBuilddir =!= null then (
